### PR TITLE
Consistent Args

### PR DESCRIPTION
**Pull request by rentainhe**
## TODO
- [x] 对齐scheduler传入的args
- [x] 更新当传入的`warmup iters=0`的时候的warning info

### CODE REVIEW

**Comment by rentainhe** [comment]
## CosineLR和CosineAnnealingLR对比
设置`max_iters=30`的时候
- CosineLR
image

- CosineAnnealingLR
image

File: libai/scheduler/lr_scheduler.py [hunk 1]
```diff
@@ -55,19 +55,19 @@ def WarmupCosineLR(
 @SCHEDULER_REGISTRY.register()
 def WarmupCosineAnnealingLR(
     optimizer: flow.optim.Optimizer,
-    t_max: int,
+    max_iters: int,
```
Comment (rentainhe) [inline libai/scheduler/lr_scheduler.py:58]: 这里统一改为`max_iters`，只是在(0, max_iters)这个区间里这个LR的曲线似乎和CosineLR是一致的，所以似乎略有重复，不过我感觉可以保留，到时候在API Reference里画图给出区别

File: libai/scheduler/lr_scheduler.py [hunk 2]
```diff
@@ -55,19 +55,19 @@ def WarmupCosineLR(
...
     **kwargs
 ):
     cosine_annealing_lr = flow.optim.lr_scheduler.CosineAnnealingLR(
-        optimizer, T_max=t_max, eta_min=eta_min
+        optimizer, T_max=max_iters, eta_min=eta_min
     )
     if warmup_iters == 0:
         logger.warning(
-            "warmup iters equals to zero, directly return the passed scheduler"
+            "warmup iters equals to zero, return CosineAnnealingLR"
```
Comment (rentainhe) [inline libai/scheduler/lr_scheduler.py:70]: 这里更新了一下warmup_iters=0的时候返回的warning，可以看看这样的格式可不可以

**Review by L1aoXingyu — APPROVED** [review]
